index.html

```diff
@@ -0,0 +1,242 @@
+    <link rel="stylesheet" href="assets/style/mainMediaqueries.css">
+    <script src="./assets/scripts/main.js" defer></script>
+    <script src="./assets/scripts/menu.js" defer></script>
+    <script src="assets/scripts/expertsList.js" defer></script>
+    <title>Home</title>
+</head>
+
+<body>
+    <div class="top-contact">
+        <ul>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <header class="main-header">
+        <nav>
+            <div class="menu-logo">
+                <div class="menu-con">
+                    <i class='bx bx-menu'></i>
+                </div>
+
+                <div class="logo">
+                    <img src="assets/images/logo.png" alt="">
+                </div>
+
+            </div>
+
+            <ul>
+                <a class="active-link" href="/">
+                    <li>Home</li>
+                </a>
+                <a href="about.html">
+                    <li>About</li>
+                </a>
+                <a href="#">
+                    <li>Join</li>
+                </a>
+                <a href="#">
+                    <li>Sponsor</li>
+                </a>
+                <a href="#">
+                    <li>News</li>
+                </a>
+            </ul>
+        </nav>
+
+        <div class="mobile-menu">
+            <div class="close-btn-container">
+                <i class='bx bx-x'></i>
+            </div>
+            <ul>
+                <a class="active-link" href="/">
+                    <li>Home</li>
+                </a>
+                <a href="about.html">
+                    <li>About</li>
+                </a>
+                <a href="#">
+                    <li>Join</li>
+                </a>
+                <a href="#">
+                    <li>Sponsor</li>
+                </a>
+                <a href="#">
+                    <li>News</li>
+                </a>
+            </ul>
+        </div>
+    </header>
+
+    <section class="hero">
+        <div class="hero-container">
+            <div class="hero-heading">
+                <h2>"Let's change your look"</h2>
+                <div class="">
+                    <h1>Enjoy the look of perfection and <br> change your style</h1>
+                </div>
+
+                <div class="hero-desc">
+                    <p>
+                        Our team of fashion experts carefully handpicks each item, ensuring that you receive the highest
+                        quality products that are both on-trend and timeless. From luxurious fabrics to meticulous
+                        craftsmanship, we prioritize delivering exceptional pieces that will elevate your wardrobe.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="events">
+        <div class="title">
+            <h2 class="">Main Program</h2>
+        </div>
```

Comment:
[Optional] Kindly update the styling of the underline under the section title to match the behance design better. I suggest you reduce the size and make it center aligned

Your design:
<img width="497" alt="image" src="https://github.com/DjibrilM/capstone-project/assets/37116322/164e334d-5dc1-4fbf-9e9c-763439f7502a">

Behance design:
<img width="626" alt="image" src="https://github.com/DjibrilM/capstone-project/assets/37116322/215b7b13-700e-452b-8b8e-d8d02c9ac63c">